roles/default/kiali-deploy/defaults/main.yml

```
@@ -207,6 +207,7 @@ kiali_defaults:
     version_label_name: "version"
 
   kiali_feature_flags:
+    clustering: true
```

I would prefer this:
```suggestion
    clustering:
      enabled: true
```